raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Access to Environmental Information
+(101) To enable public participation, there shall be freedom of access to environmental information: [1] Any person who desires to obtain information relating to the implementation of this Act or any other information concerning the development and management of the environment and natural resources in accordance with this Law shall have freedom of access to that information; [2] A personae requesting to obtain the information shall apply to the Agency in a prescribed form indication the type of information required and may pay a minimal fee which may be prescribed by the Agency; [3] Freedom of access to information; [a] does not extend to proprietary information, which upon application by proprietor for protection shall be treated by the Agency as confidential; [b] and provided that the application for protection does not relate to such information as emission data and does not otherwise defeat the principles and objective of this Law. [4] A person requesting to obtain the information shall apply to the Agency in a prescribed form indicating the type of information required and may pay a minimal or reasonable fee which may be prescribed by the Agency; [5] For the purposes of facilitating public participation and access to information relating to the management of the environment and the state of natural resources, the Agency shall establish and operate a Public Registry within its Offices; [6] In addition to a Public Registry within its offices; the Agency may establish and operate County Public Registries and Libraries within its offices at the County; [7] The public registry shall contain all records and information produced, collected, or submitted in accordance with the requirements of this Law; [8] Notwithstanding any law or regulations of Liberia, no civil or criminal proceedings shall lie against the Agency and Line Ministries, or its officers or any person acting on behalf of or under the direction of the Agency or Ministry, for the disclosure of any record pursuant of this Law.
+
+Environmental Education
+(102) The Agency shall, in consultation with the Ministry of Education take appropriate measures for the integration in schools, colleges and university curricula of environment education.
+
+State of the Environment Report
+(103) [1] The Agency shall every 5 years publish the Report on the State of the Environment of Liberia; [2] The Report shall in addition to other matters as may be prescribed, specify the main activities of the Agency and Line Ministries regarding the protection of the environment; [3] The Report shall take into account the plans and activities of the County and District Committees established under this Law; [4] The Agency may, publish any other information it considers necessary for information and public education on the environment.
```

Comment:
Please replace each semi-colon preceding a number in square brackets with a period. Also, please replace `The Agency may, publish` with `The Agency may publish`.